scripts/check_edge_docs.py

```
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)